hoodie-common/src/main/java/com/uber/hoodie/common/table/TableFileSystemView.java

```diff
@@ -29,6 +33,7 @@
  * <p>
  * ReadOptimizedView - Lets queries run only on organized columnar data files at the expense of latency
  * WriteOptimizedView - Lets queries run on columnar data as well as delta files (sequential) at the expense of query execution time
+ *
  * @since 0.3.0
  */
 public interface TableFileSystemView {
```

Comment:
Not sure if all FileSystemView will share an interface.. Atleast in my head, FileSystemView => real time, read optimized , provide different operations..

for eg: ReadOptimized view does not offer groupLatestDataFileWithLogFiles at all